google/logging/v2/logging_gapic.yaml

```diff
@@ -352,8 +352,11 @@ interfaces:
     bundling:
       thresholds:
         element_count_threshold: 100
-        request_byte_threshold: 1024 # 1 Kb
+        request_byte_threshold: 20480 # 20 Kb
```

Comment:
Why does this affect flow control and not `request_byte_threshold`? The 5MB limit is for a single request (i.e. a single bundle), which is what `request_byte_threshold` will affect. The flow control setting will affect the total number of bytes that can be queued in the bundler before additional calls either block or throw an exception. So the `request_byte_threshold` could be 4MB and the flow control setting could be 40MB, which would mean a max of approx 10 max size bundles could be queued.